contracts/truefi2/PoolFactory.sol

```diff
@@ -120,7 +130,7 @@ contract PoolFactory is IPoolFactory, UpgradeableClaimable {
         pool[token] = address(proxy);
         isPool[address(proxy)] = true;
 
-        ITrueFiPool2(address(proxy)).initialize(ERC20(token), liquidationToken, trueLender2, ONE_INCH_ADDRESS, this.owner());
+        ITrueFiPool2(address(proxy)).initialize(ERC20(token), liquidationToken, trueLender2, ONE_INCH_ADDRESS, safu, this.owner());
```

Comment:
If someone creates a new pool before we set safu in this factory, safu will be set to 0. Maybe let's require it not to be 0?

Reply:
We have a setter for this case, it's fine

Reply:
This is fine 🔥🐶🔥

Reply:
I think safu should stay out of initializer and should be set by setter. Not all of the pools will be guarded by the assurance fund